frontend/src/pages/learn/index.tsx

```diff
@@ -247,9 +265,18 @@ const Learn: FC = () => {
                     <h2 className="mb-8 font-display text-2xl font-medium md:mb-16 md:rounded-3xl md:bg-[#f5f5f5] md:p-6 md:text-4xl md:text-primary-700">
                       {t('quiz.navigation.title')}
                     </h2>
-                    <div className="mb-5">{renderComponent()}</div>
+                    <div>{renderComponent()}</div>
+                  </DialogContent>
+                  <DialogActions className="block"
+                  sx={{
+                    position: 'absolute',
+                    bottom: 0,
+                    left: 0,
+                    right: 0,
+                    p: 4,
+                  }}>
                     {(currentStepIndex ?? 0) > 0 && (
-                      <>
+                      <div className="">
```

Comment:
Use React Fragment shorthand insteadl.
https://react.dev/reference/react/Fragment

```suggestion
                      <>
```

Reply:
Fixed